src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -393,6 +394,14 @@ private CreationResultStatus EnterMaintenanceFlow()
                     foreach (string value in _settingsLoader.InstallUnitDescriptorCache.InstalledItems.Values)
                     {
                         Console.WriteLine($" {value}");
+
+                        if (_settingsLoader.InstallUnitDescriptorCache.Descriptors.TryGetValue(value, out IInstallUnitDescriptor descriptor))
```

Comment:
Nit: For more compact output, do we want the version to display on the same line as the pack name?

Reply:
That was my first thought, but having that line be reserved for the uninstall string and an indented section below for any details about that install descriptor being below it made it more clear that it was details about that item than part of the command